[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_unsubscribe_all_from_few_content_topics with id: 2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.124.138
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['30685', '30686', '30687', '30688', '30689']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 30685:30685 -p 30686:30686 -p 30687:30687 -p 30688:30688 -p 30689:30689 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=30687 --rest-port=30685 --tcp-port=30686 --discv5-udp-port=30688 --rest-address=0.0.0.0 --nat=extip:172.18.124.138 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=65c0bab0d02c7e1ea5a2c5aedf1ee5decf5a63d5ddb4ab4f3abcdbabd944bb33 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=30689 --metrics-logging=true --relay=true --filter=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.124.138 waku d719b70e527c2d26470be9c49fd8bd7903f39897816cfa2ee034bd93979455ee
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID d719b70e527c. Setting up logs at ./log/docker/node1_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 30685
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 068a36568f2a. Exiting log stream.
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:30685/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:30685/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.124.138/tcp/30686/p2p/16Uiu2HAm8m5FG2rhKp5QtKn7p6ftzyEQpmgLG6CTu9ZojyUVY4Zd","/ip4/172.18.124.138/tcp/30687/ws/p2p/16Uiu2HAm8m5FG2rhKp5QtKn7p6ftzyEQpmgLG6CTu9ZojyUVY4Zd"],"enrUri":"enr:-L24QIT0wtzCm1Dzu52X36QLnHXKVZjHTQcoFx0_Ig4g0hhTNZ5tO13ENKriY-YWD_WsDyULKsDPNadI4zUeCDb1Zx0CgmlkgnY0gmlwhKwSfIqKbXVsdGlhZGRyc5YACASsEnyKBnfeAAoErBJ8igZ3390DgnJzhQADAQAAiXNlY3AyNTZrMaECxiBF6t67kgPvVqXDjc3xYk4BiiD5iYpgwy920SRPFEiDdGNwgnfeg3VkcIJ34IV3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/go-waku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.49.211
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['4885', '4886', '4887', '4888', '4889']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 4885:4885 -p 4886:4886 -p 4887:4887 -p 4888:4888 -p 4889:4889 wakuorg/go-waku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=DEBUG --rest-relay-cache-capacity=100 --websocket-port=4887 --rest-port=4885 --tcp-port=4886 --discv5-udp-port=4888 --rest-address=0.0.0.0 --nat=extip:172.18.49.211 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=0b4ceef6a274576d906caf0dace5bc7f186d09ea93863955b226ac64e92c2403 --min-relay-peers-to-publish=1 --rest-filter-cache-capacity=50 --peer-store-capacity=10 --relay=false --discv5-bootstrap-node=enr:-L24QIT0wtzCm1Dzu52X36QLnHXKVZjHTQcoFx0_Ig4g0hhTNZ5tO13ENKriY-YWD_WsDyULKsDPNadI4zUeCDb1Zx0CgmlkgnY0gmlwhKwSfIqKbXVsdGlhZGRyc5YACASsEnyKBnfeAAoErBJ8igZ3390DgnJzhQADAQAAiXNlY3AyNTZrMaECxiBF6t67kgPvVqXDjc3xYk4BiiD5iYpgwy920SRPFEiDdGNwgnfeg3VkcIJ34IV3YWt1MgU --filternode=/ip4/172.18.124.138/tcp/30686/p2p/16Uiu2HAm8m5FG2rhKp5QtKn7p6ftzyEQpmgLG6CTu9ZojyUVY4Zd
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.49.211 waku 7869245b83f30b6b1ab0ab18d485d1833ae56d97e3f834fa695dd8b938a027f1
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 7869245b83f3. Setting up logs at ./log/docker/node2_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/go-waku:latest. REST: 4885
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:4885/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"enrUri":"enr:-J-4QBpnL0sL_OTHptECLVN-ZQC53P7Nv_RPS8X9qJetbYFFDITEntILiSNLPKUT6okxywu6s94klECHiCXlQ4jMN9eGAZNMzDwVgmlkgnY0gmlwhKwRAAOCcnODAAMAiXNlY3AyNTZrMaEDgb-mBpp1ARFon2yRAVFnbEGTHzm-pknhGy5swLobeZiDdGNwghMWg3VkcIITGIV3YWt1MoA","listenAddresses":["/ip4/127.0.0.1/tcp/4886/p2p/16Uiu2HAmMPUSPdVZfh2JEndCw95NmUisz1KzFXnT6uYdSkWJA6Th","/ip4/127.0.0.1/tcp/4887/ws/p2p/16Uiu2HAmMPUSPdVZfh2JEndCw95NmUisz1KzFXnT6uYdSkWJA6Th","/ip4/172.17.0.3/tcp/4886/p2p/16Uiu2HAmMPUSPdVZfh2JEndCw95NmUisz1KzFXnT6uYdSkWJA6Th","/ip4/172.17.0.3/tcp/4887/ws/p2p/16Uiu2HAmMPUSPdVZfh2JEndCw95NmUisz1KzFXnT6uYdSkWJA6Th"]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.124.138/tcp/30686/p2p/16Uiu2HAm8m5FG2rhKp5QtKn7p6ftzyEQpmgLG6CTu9ZojyUVY4Zd", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:30685/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:4885/filter/v2/subscriptions. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:4885/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "74af3dfc-3e41-4ebe-9c71-27bd263d62b1", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"74af3dfc-3e41-4ebe-9c71-27bd263d62b1","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:30685/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "Hello World!", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/go-waku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:4885/filter/v2/messages/%2Fwaku%2F2%2Frs%2F3%2F1/Hello%20World%21" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id d719b70e527c
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 7869245b83f3
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container d719b70e527c. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m tests.conftest:conftest.py:73 Running fixture teardown: attach_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:74 Test failed, attempting to attach logs to the allure reports
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node1_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node2_2024-11-21_03-37-51__43f39af6-65d8-45af-bbb5-38c51e6e7ee0__wakuorg_go-waku:latest.log
[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_unsubscribe_all_from_few_content_topics with id: 2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.73.16
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['64573', '64574', '64575', '64576', '64577']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 64573:64573 -p 64574:64574 -p 64575:64575 -p 64576:64576 -p 64577:64577 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=64575 --rest-port=64573 --tcp-port=64574 --discv5-udp-port=64576 --rest-address=0.0.0.0 --nat=extip:172.18.73.16 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=14c9d13a78b2ee219c192dda39b4367acdba8bdc9abec1d1dcdd3ade8462dcd1 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=64577 --metrics-logging=true --relay=true --filter=true
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 7869245b83f3. Exiting log stream.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.73.16 waku 44e86c72ede42aa9f1515c3a9d1f8666044a2025a8f9ad25eca564fa221e1dcb
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 44e86c72ede4. Setting up logs at ./log/docker/node1_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 64573
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64573/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64573/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.73.16/tcp/64574/p2p/16Uiu2HAmQFUf6T32F8hyFKqxLFHNCgHbZG3DnDZMTx879PBi8zxJ","/ip4/172.18.73.16/tcp/64575/ws/p2p/16Uiu2HAmQFUf6T32F8hyFKqxLFHNCgHbZG3DnDZMTx879PBi8zxJ"],"enrUri":"enr:-L24QAd6hYYKmb_dRQNaB3F34u8p7eHTP1YOIyOFShKpTKKfXFMqXPqy3Wx-wYJJbqkW7TPQC4utOk3S94vqo_QUrBwCgmlkgnY0gmlwhKwSSRCKbXVsdGlhZGRyc5YACASsEkkQBvw-AAoErBJJEAb8P90DgnJzhQADAQAAiXNlY3AyNTZrMaEDrEZPfSr3O_PiARVTMXDUvArbcnAIHURTLxv1YnkKZfODdGNwgvw-g3VkcIL8QIV3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/go-waku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.252.32
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['46387', '46388', '46389', '46390', '46391']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 46387:46387 -p 46388:46388 -p 46389:46389 -p 46390:46390 -p 46391:46391 wakuorg/go-waku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=DEBUG --rest-relay-cache-capacity=100 --websocket-port=46389 --rest-port=46387 --tcp-port=46388 --discv5-udp-port=46390 --rest-address=0.0.0.0 --nat=extip:172.18.252.32 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=a02dabdffea7fdf0dbf23c2836c58fceeeb2f7ecffb9c81deaef7db25c52388b --min-relay-peers-to-publish=1 --rest-filter-cache-capacity=50 --peer-store-capacity=10 --relay=false --discv5-bootstrap-node=enr:-L24QAd6hYYKmb_dRQNaB3F34u8p7eHTP1YOIyOFShKpTKKfXFMqXPqy3Wx-wYJJbqkW7TPQC4utOk3S94vqo_QUrBwCgmlkgnY0gmlwhKwSSRCKbXVsdGlhZGRyc5YACASsEkkQBvw-AAoErBJJEAb8P90DgnJzhQADAQAAiXNlY3AyNTZrMaEDrEZPfSr3O_PiARVTMXDUvArbcnAIHURTLxv1YnkKZfODdGNwgvw-g3VkcIL8QIV3YWt1MgU --filternode=/ip4/172.18.73.16/tcp/64574/p2p/16Uiu2HAmQFUf6T32F8hyFKqxLFHNCgHbZG3DnDZMTx879PBi8zxJ
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.252.32 waku 478600e0f5db0a8448aa0515a129d7f010329b0b058cbbe97e3ebc84f14f3a2e
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 478600e0f5db. Setting up logs at ./log/docker/node2_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/go-waku:latest. REST: 46387
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46387/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"enrUri":"enr:-J-4QJaB9xzDJE_ZbxVt1-6TAWXqvt8R3ULOmHkFE-azg4DcfhUBNq1XfQTaOCpEvTVa36mv-OxnGnT4Ie-jnfy_oe-GAZNMzPc9gmlkgnY0gmlwhKwRAAWCcnODAAMAiXNlY3AyNTZrMaECHhIGWYcMGe62UA7cY55gE7GxWFva-0TYLzsdY-xdf72DdGNwgrU0g3VkcIK1NoV3YWt1MoA","listenAddresses":["/ip4/127.0.0.1/tcp/46388/p2p/16Uiu2HAkwT49n8E1cWQG4NbWBvsCD1AGnJWZkPutcMnKjXRTxzgk","/ip4/127.0.0.1/tcp/46389/ws/p2p/16Uiu2HAkwT49n8E1cWQG4NbWBvsCD1AGnJWZkPutcMnKjXRTxzgk","/ip4/172.17.0.5/tcp/46388/p2p/16Uiu2HAkwT49n8E1cWQG4NbWBvsCD1AGnJWZkPutcMnKjXRTxzgk","/ip4/172.17.0.5/tcp/46389/ws/p2p/16Uiu2HAkwT49n8E1cWQG4NbWBvsCD1AGnJWZkPutcMnKjXRTxzgk"]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.73.16/tcp/64574/p2p/16Uiu2HAmQFUf6T32F8hyFKqxLFHNCgHbZG3DnDZMTx879PBi8zxJ", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:64573/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:46387/filter/v2/subscriptions. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"subscription failed"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46387/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "36dfbb25-ae77-4c57-baef-663799aa97bf", "contentFilters": ["Hello World!", "1234567890", "{\"key\": \"value\"}", "\u8fd9\u662f\u4e00\u4e9b\u4e2d\u6587", "\ud83d\ude80\ud83c\udf1f\u2728"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"36dfbb25-ae77-4c57-baef-663799aa97bf","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:64573/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "Hello World!", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/go-waku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46387/filter/v2/messages/%2Fwaku%2F2%2Frs%2F3%2F1/Hello%20World%21" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 44e86c72ede4
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 478600e0f5db
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 44e86c72ede4. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m tests.conftest:conftest.py:73 Running fixture teardown: attach_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:74 Test failed, attempting to attach logs to the allure reports
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node1_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node2_2024-11-21_03-38-38__9e064527-f280-4258-9866-802379d39422__wakuorg_go-waku:latest.log